internal/backup_fetch_handler_new.go

```diff
@@ -28,38 +83,41 @@ func GetPgFetcherNew(dbDataDirectory, fileMask, restoreSpecPath string) func(fol
 			tracelog.ErrorLogger.FatalfOnError("Failed to fetch backup: %v\n",
 				newNonEmptyDbDataDirectoryError(dbDataDirectory))
 		}
-
-		err = deltaFetchRecursionNew(backup.Name, folder, utility.ResolveSymlink(dbDataDirectory), spec, filesToUnwrap)
```

Comment:
The bad side is dependencies are hidden with the config.

Reply:
The good side is that was easy to read how these variables were modified with applyUnwrapResult.
Take decision by yourself. Locate variables to config or not.